drivers/ethernet/eth_dwmac_stm32h7x.c

```diff
@@ -57,8 +58,7 @@ int dwmac_bus_init(struct dwmac_priv *p)
 		return -EIO;
 	}
 
-	ret = stm32_dt_pinctrl_configure(eth0_pins, ARRAY_SIZE(eth0_pins),
-					 (uint32_t)p->base_addr);
+	ret = pinctrl_apply_state(eth0_pins, PINCTRL_STATE_DEFAULT);
```

Comment:
nit: probably adjust the variable name here, `pcfg`?

Reply:
Makes sense. Will update